docs/Makefile

```diff
@@ -34,6 +34,9 @@ html:
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+html-rustdoc:
+	cd ../rust && cargo rustdoc --features untrusted,derive -- --html-in-header opendp_tooling/katex.html --document-private-items
```

Comment:
Don't you already specify the katex thing in `Cargo.toml`?

Reply:
That makes it the html apply when rendered on docs.rs only, unfortunately.

Reply:
I missed that, thanks!